src/status_im/multiaccounts/key_storage/core.cljs

```diff
@@ -264,6 +266,16 @@ We don't need to take the exact steps, just set the required state and redirect
             {:db (assoc db :goto-key-storage? true)}
             (multiaccounts.logout/logout)))
 
+(rf/defn open-device-settings-dialog
+  {:events [::open-device-settings-dialog]}
+  [_]
+  {:ui/show-confirmation
+   {:title               (i18n/label :t/allow-access)
+    :content             (i18n/label :t/camera-access-description)
+    :confirm-button-text (i18n/label :t/settings)
+    :on-accept           #(.openSettings ^js react/linking)
```

Comment:
could you elaborate why `openSettings` is needed

Reply:
![Simulator Screen Shot - iPhone 13 - 2023-06-23 at 11 01 20](https://github.com/status-im/status-mobile/assets/15260284/d7d0c1ff-1504-450d-a015-beb1c7a2c79d)
Need to open the setting page as per figma

Reply:
thanks